src/Db/Dump/PKSimDB_dump.txt

```diff
@@ -85812,10 +85812,10 @@ INSERT INTO tab_rate_generic_molecules VALUES('LinksCommon','MucosaPgPToLumen_MM
 INSERT INTO tab_rate_generic_molecules VALUES('LinksCommon','MucosaPgPToLumen_MM',366,'DRUG','TM',1);
 INSERT INTO tab_rate_generic_molecules VALUES('LinksCommon','PgpSpecific_Hill',341,'DRUG','C_cell',0);
 INSERT INTO tab_rate_generic_molecules VALUES('LinksCommon','PgpSpecific_Hill',351,'DRUG','C_int',0);
-INSERT INTO tab_rate_generic_molecules VALUES('LinksCommon','PgpSpecific_Hill',366,'DRUG','TM',1);
+INSERT INTO tab_rate_generic_molecules VALUES('LinksCommon','PgpSpecific_Hill',368,'DRUG','TM',1);
 INSERT INTO tab_rate_generic_molecules VALUES('LinksCommon','PgpSpecific_MM',341,'DRUG','C_cell',0);
 INSERT INTO tab_rate_generic_molecules VALUES('LinksCommon','PgpSpecific_MM',351,'DRUG','C_int',0);
-INSERT INTO tab_rate_generic_molecules VALUES('LinksCommon','PgpSpecific_MM',366,'DRUG','TM',1);
+INSERT INTO tab_rate_generic_molecules VALUES('LinksCommon','PgpSpecific_MM',368,'DRUG','TM',1);

```

Comment:
For `BrainPgPToPlasma_Hill` and `BrainPgPToPlasma_MM` must be changed from SOURCE|TRANSPORTER to TARGET|TRANSPORTER as well.

(`MucosaPgPToLumen_Hill` and `MucosaPgPToLumen_MM` are OK, there it's really SOURCE)

Reply:
done